src/test/java/org/broadinstitute/hellbender/tools/walkers/RevertBaseQualityScoresIntegrationTest.java

```diff
@@ -57,4 +56,20 @@ public void testNoOriginalQuals() throws IOException {
         };
         runCommandLine(args);
     }
-}
\ No newline at end of file
+
+    //Regression test for https://github.com/broadinstitute/gatk/issues/1473
+    @Test
+    public void testBadReads() throws IOException {
+        final File inputBam = new File(getTestDataDir(), "badRead.sam");
```

Comment:
this should mention somewhere why this read is bad.  maybe `testReadThatsEntirelyInsertion`